spark-plugins/pom.xml

```diff
@@ -193,6 +193,47 @@
           </execution>
         </executions>
       </plugin>
+	  <plugin>
+      <groupId>org.apache.rat</groupId>
+      <artifactId>apache-rat-plugin</artifactId>
+      <version>0.10</version>
+        <executions>
+          <execution>
+            <id>rat-check</id>
+            <phase>validate</phase>
+            <goals>
+              <goal>check</goal>
+            </goals>
+            <configuration>
```

Comment:
This is done to exclude rat check for "test resources" folder which contains "testData.csv" used for training the data.
Please let me know if the exclusion can be mentioned at the root pom level to exclude all resources, or is it okay to keep it specific to this case.